internal/aws.go

```diff
@@ -117,6 +120,9 @@ func assumeRoleFromAWS(arn, externalID string, request *Request) (*sts.AssumeRol
 	span.SetTag("aws.arn", arn)
 	span.SetTag("aws.external_id", externalID)
 
+	roleDurationSeconds := getenvDefault("ROLE_DURATION_SECONDS", "3600")
+	intRoleDurationSeconds, _ := strconv.ParseInt(roleDurationSeconds, 10, 64)
```

Comment:
we should validate and read the environment on startup, not on each request - and please handle the error :)